application/views/invoice_templates/pdf/InvoicePlane - overdue.php

```diff
@@ -30,28 +30,34 @@
         if ($invoice->client_address_2) {
             echo '<div>' . htmlsc($invoice->client_address_2) . '</div>';
         }
-        if ($invoice->client_city || $invoice->client_state || $invoice->client_zip) {
```

Comment:
Is not it better to leave this configuration by default? I think it's the most logical.

Reply:
Well it's just the question which standard should be used. The current one is for north america (stete before zip). This matches the default system configuration for amounts and so on so I don't see a need to change anything here.
This will change completely with InvoicePlane 2 so I think changes are not necessary here. Also, it may confuse existing users as many use the standard templates.

Reply:
perfect ;)